bom/pom.xml

```diff
@@ -935,6 +942,27 @@
         <artifactId>cryptvault</artifactId>
         <version>${cryptvault.version}</version>
       </dependency>
+      <dependency>
+        <!-- Only needed for the org.apache.logging.log4j.kubernetes.ContainerUtil class for now, hence the kubernetes-client exclusion;
```

Comment:
if we only need the *ContainerUtil* class then we can also simply copy the code into our code base and keep the copyright header. It feels a little awkward to introduce this dependency which seems to be about logging which we do not (want to) use ...

Reply:
Well, the added dependency with the exclusions for essentially just one method is indeed looking a bit awkward. On the other hand, I'm not a fan of copying classes if it can be avoided. So, pros and cons. I'm also ok with copying the class.

I've updated the PR accordingly, also adding a chapter in the `NOTICE.md`, along with a note in the copied class. @sophokles73 Is that sufficient from a license perspective?